video_xblock/video_xblock.py

```diff
@@ -233,6 +233,20 @@ def save_player_state(self, request, suffix=''):
         self.player_state = player_state
         return {'success': True}
 
+    @XBlock.json_handler
+    def publish_event(self, data, suffix=''):
+        """
+        Handler to publish XBlock event from frontend.
+        Called by student_view's JavaScript
+        """
+        try:
+            event_type = data.pop('event_type')
```

Comment:
CopyPaste from 
https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/master/drag_and_drop_v2/drag_and_drop_v2.py#L447